astroquery/gaia/tests/test_gaiatap.py

```diff
@@ -16,6 +16,8 @@
 """
 import unittest
 import os
+import pytest
```

Comment:
Ah, yes, I forgot about that environment variable. Still, it appears confusing. But I am glad you already have plans to address it.